src/helpers/index.js

```diff
@@ -0,0 +1,10 @@
+import './questions-manager';
+import './piping';
+import './previous-link';
+import SummaryManager from './summary-manager';
+
+const summaryPlaceholder = document.querySelector('.js-summary');
```

Comment:
```suggestion
const summaryPlaceholder = document.querySelector('.ons-js-summary');
```

Reply:
@jrbarnes9 We don't need to do this as it's just for the prototype kit.